modules/openy_memberships_front/openy_memberships_front.libraries.yml

```diff
@@ -0,0 +1,15 @@
+openy_memberships_front:
+  js:
```

Comment:
let's provide default versions everywhere. Given the fact this is a first version - 0.1 should be fine. Better caching control if defined